src/components/access-type-display/access-type-display.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { KeyValue, NoValue } from '@folio/stripes/components';
+
+import { getAccessTypeId } from '../utilities';
+
+const propTypes = {
+  accessStatusTypes: PropTypes.object.isRequired,
+  model: PropTypes.object.isRequired,
+};
+
+function AccessTypeDisplay({ model, accessStatusTypes }) {
+  const accessTypesRecords = accessStatusTypes.resolver.state.accessTypes.records;
```

Comment:
please, pass already formatted data, so that dumb component like `AccessTypeDisplay` don't have to care about things like `resolvers`